nodes/parachain/src/command.rs

```diff
@@ -121,7 +121,7 @@ pub(crate) fn run() -> sc_cli::Result<()> {
 			})
 		}
 		Some(Subcommand::ExportGenesisState(cmd)) => {
-			let (chain_spec_id, runtime) = get_selected_chainspec(&cli)?;
+			let (chain_spec_id, runtime) = get_selected_chainspec(&cmd.shared_params)?;
 			let spec = cli.load_spec(chain_spec_id.as_str())?;
 
 			println!("Dispatching task for spec id: {chain_spec_id}.");
```

Comment:
There is this print call too, not sure if it should be removed.

Reply:
It was already there. I just removed the debug prints that I needed to troubleshoot the issue. So, I think it's fine now.